automation-roles/60-configure-cloud-pak/cp4d/cp4d-cluster-configure/tasks/main.yml

```diff
@@ -9,6 +9,13 @@
     query: >-
       [?name=='{{ current_cp4d_cluster.openshift_cluster_name }}']
 
+- set_fact:
+    _cpd_accept_licenses: "False"
+
+- set_fact:
+    _cpd_accept_licenses: "True"
+  when: (cpd_accept_licenses | default(False) | bool) or (_current_openshift_cluster.accept_licenses | default(False) | bool)
+
```

Comment:
I think here you also need to fail if the license was not accepted. This in case the deployer was run with --cp-config-only

Reply:
Agreed, let me make that change

Reply:
@fketelaars check is implemented